doc/language.md

```diff
@@ -801,6 +801,14 @@ this list, but most apps can safely use them:
   As a special exception to this rule, `List(Bool)` may **not** be upgraded to a list of structs,
   because implementing this for bit lists has proven unreasonably expensive.
 
+* A integer field may be promoted to an integer field of larger width while keeping the sign
+* A boolean field may be promoted to an integer field
```

Comment:
Not compatible: Similar to integers. If you have two boolean fields, they will be packed in adjacent bits. If you change the first one to an integer, the second one will move.